src/transformers/modeling_tf_utils.py

```diff
@@ -2136,6 +2137,8 @@ def save_pretrained(
                 Whether or not to push your model to the Hugging Face model hub after saving it. You can specify the
                 repository you want to push to with `repo_id` (will default to the name of `save_directory` in your
                 namespace).
+            signatures (`dict` or `tf.function`, *optional*, defaults to `None`):
```

Comment:
```suggestion
            signatures (`dict` or `tf.function`, *optional*):
```
We don't indicate the default when it's None.